amgut/static/js/interactive.js

```diff
@@ -0,0 +1,350 @@
+$(document).ready(function() {
```

Comment:
One general comment in this file is that there's documentation missing for all these functions, I think it would be worth adding that right now.